rules/sdl2.json

```diff
@@ -0,0 +1,56 @@
+{
+  "patterns": ["\\bsdl2\\b"],
+  "dependencies": [
+    {
+      "packages": ["libsdl2-dev"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "ubuntu"
+        },
+        {
+          "os": "linux",
+          "distribution": "debian"
+        }
+      ]
+    },
+    {
+      "packages": ["SDL2-devel"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "centos"
+        },
+        {
+          "os": "linux",
+          "distribution": "rockylinux"
+        },
+        {
+          "os": "linux",
+          "distribution": "redhat"
+        },
+        {
+          "os": "linux",
+          "distribution": "fedora"
+        },
+        {
+          "os": "linux",
+          "distribution": "opensuse"
+        },
+        {
+          "os": "linux",
+          "distribution": "sle"
```

Comment:
Thanks! There were CI failures with 3 distros, and it looks like SDL2 is available on each but will need minor adjustments:
- On openSUSE/SLES, the package name is different, possibly `libSDL2-devel` but definitely double check since there are a few different packages.
- On CentOS/RHEL 7, SDL2 needs EPEL installed first (see GDAL for example: https://github.com/rstudio/r-system-requirements/blob/main/rules/gdal.json)
- On CentOS/RHEL 8, SDL2 needs EPEL and also PowerTools (CentOS/Rocky 8) or CodeReady Builder (RHEL 8). See GDAL for an example as well

It'd also be fine to exclude any of these distros for now, especially RHEL 7 which is EOL soon.

Reply:
Thanks @glin for your feedback. I've updated accordingly, I also ran the actions and now it is passing. The only problem is something about `google-chrome-stable` from `centos 7`, but that seems to be an unrelated error which is affecting all the `centos 7` CI's though.